client/src/App.js

```diff
@@ -1,9 +1,41 @@
+import { useEffect, useState } from 'react';
 import Ohm from './pages/Ohm';
+import ExercisesCategory from './components/ExercisesCategory';
+import BodyParts from './pages/BodyParts';
+import { Route, Switch } from 'react-router-dom';
```

Comment:
Structure your imports from external stuff to your internal components. Also group pages and components together. And ideally alphabetically 👍 
```suggestion
import { useEffect, useState } from 'react';
import { Route, Switch } from 'react-router-dom';
import BodyParts from './pages/BodyParts';
import Ohm from './pages/Ohm';
import ExercisesCategory from './components/ExercisesCategory';
```